subgraph/deploy-develop.sh

```diff
@@ -1,4 +1,4 @@
-export IPFS_URL="http://localhost:5001"
+export DEVELOP_IPFS_URI="https://api.thegraph.com/ipfs/"
```

Comment:
Okay. I'll leave @bussyjd comment the approach here, but It's contextually meaningful.

Reply:
Already validated with @lucarlig :+1: